### PR TITLE
Optional theme for angular

### LINKED ISSUE #272: Optional theme support for ignteiui for angular projects.

### PR DESCRIPTION
**Pull request by wnvko**
Closes #272

Default theme for Angular projects added. To retain current state by default Custom theme is chosen.

### CODE REVIEW

File: templates/angular/igx-ts/projects/empty/files/angular.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+			  "src/styles.scss"$(DefaultTheme)
```
Comment (damyanpetev) [inline templates/angular/igx-ts/projects/empty/files/angular.json]: Indent formatting is off. Would prefer if it remains consistent.

File: templates/angular/igx-ts/projects/empty/index.ts [hunk 2]
```diff
@@ -10,6 +10,21 @@ class EmptyJsBlocksProject implements ProjectTemplate {
...
+$primary: #731963 !default;
+$secondary: #ce5712 !default;
+
+$app-palette: igx-palette($primary, $secondary);
+
+@include igx-core();
+@include igx-theme($app-palette);
+	`;
+	private DEFAULT_THEME = `,
+			  "node_modules/igniteui-angular/styles/igniteui-angular.css"`;
```
Comment (damyanpetev) [inline templates/angular/igx-ts/projects/empty/index.ts]: The tabs in this string will also mess the formatting.

File: spec/unit/PromptSession-spec.ts [hunk 3]
```diff
@@ -435,4 +435,51 @@ describe("Unit - PromptSession", () => {
...
+		await mockSession.start();
+		const CUSTOM_THEME = `
+		/* See: https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes.html */
+		@import "~igniteui-angular/core/styles/themes/index";
+		$primary: #731963 !default;
+		$secondary: #ce5712 !default;
+		$app-palette: igx-palette($primary, $secondary);
+		@include igx-core();
+		@include igx-theme($app-palette);`;
+		const actualCall = (Util.processTemplates as any).calls.argsFor(0)[2]["$(CustomTheme)"].replace(/\s/g, "");
```
Comment (damyanpetev) [inline spec/unit/PromptSession-spec.ts]: `as jasmine.Spy` if you want the intellisense :)

File: templates/angular/igx-ts/projects/empty/index.ts (hunk 2 above)
Comment (wnvko) [reply templates/angular/igx-ts/projects/empty/index.ts]: Done

File: templates/angular/igx-ts/projects/empty/files/angular.json (hunk 1 above)
Comment (wnvko) [reply templates/angular/igx-ts/projects/empty/files/angular.json]: Done

**Pull request merged**